Vizzuality/landgriffon
# Marketing-v2 changes in img, video and minor styles

### General description

_Please write a description. If the PR is hard to understand, provide a quick explanation of the code._

- **About**: images change + no autoplay in video in mobile screens.
- **Home**: Image has a white line on the bottom - had to cut the img and reupload. 
- **The service**: image border blur was too much so toned it down a bit. 
- **Methodology**: logos had too much space between them.
- 
### Designs

_Link to the related design prototypes (if applicable)_

### Testing instructions

For testing just double check that all those changes are in the pages of the list.

- Apart from the added feature / bug fix, check overall performance, styling...

## Checklist before merging
